('/home/sudosom/build/usbGPUdriver/usbGPUdriver',
 True,
 False,
 True,
 None,
 None,
 False,
 False,
 None,
 True,
 False,
 None,
 None,
 None,
 '/home/sudosom/build/usbGPUdriver/usbGPUdriver.pkg',
 [('pyi-contents-directory _internal', '', 'OPTION'),
  ('PYZ-00.pyz', '/home/sudosom/build/usbGPUdriver/PYZ-00.pyz', 'PYZ'),
  ('struct',
   '/home/sudosom/build/usbGPUdriver/localpycs/struct.pyc',
   'PYMODULE'),
  ('pyimod01_archive',
   '/home/sudosom/build/usbGPUdriver/localpycs/pyimod01_archive.pyc',
   'PYMODULE'),
  ('pyimod02_importers',
   '/home/sudosom/build/usbGPUdriver/localpycs/pyimod02_importers.pyc',
   'PYMODULE'),
  ('pyimod03_ctypes',
   '/home/sudosom/build/usbGPUdriver/localpycs/pyimod03_ctypes.pyc',
   'PYMODULE'),
  ('pyiboot01_bootstrap',
   '/home/sudosom/.local/lib/python3.11/site-packages/PyInstaller/loader/pyiboot01_bootstrap.py',
   'PYSOURCE'),
  ('pyi_rth_inspect',
   '/home/sudosom/.local/lib/python3.11/site-packages/PyInstaller/hooks/rthooks/pyi_rth_inspect.py',
   'PYSOURCE'),
  ('usbGPUdriver', '/home/sudosom/usbGPUdriver.py', 'PYSOURCE')],
 [],
 False,
 False,
 1719652139,
 [('run',
   '/home/sudosom/.local/lib/python3.11/site-packages/PyInstaller/bootloader/Linux-64bit-arm/run',
   'EXECUTABLE')],
 '/lib/aarch64-linux-gnu/libpython3.11.so')
